exercises/roman-numerals/src/example.c

```diff
@@ -0,0 +1,43 @@
+#include "roman_numerals.h"
+#include <stdlib.h>
+#include <string.h>
+
+#define NUM_OF_ELEMENTS(a)    (sizeof(a) / sizeof(a[0]))
+#define MAX_NUMERAL_LENGTH    (6)
```

Comment:
Isn't it more than that? "MMMDCCCLXXXVIII" is the longest valid answer I can think of at the moment.